### DISCUSSION TITLE
defineEventHandler(event) supplies different request header format on development and production builds

### DISCUSSION THREAD

**Discussion by rickgregg** [nuxt/nuxt#24220] [Questions]
defineEventHandler(event) supplies different request Header format on development and production builds. This can cause a problem when searching for a Header name in the event handler code.

Versions Used
============
Nuxt version: 3.7.3
Nitro version 2.6.3
npm version: 9.6.6
vite version: 4.4.9

Steps to Recreate
===========

1. Write and invoke the following simple event handler

```
export default defineEventHandler(async (event) => {
….
const rawHeaders = event.node.req.rawHeaders;
console.log(‘rawHeaders: ‘, rawHeaders);
…..
}
```

2. npm run dev - Each word of the Header name is in the lowercase format (i.e. sec-fetch-mode)
3. npm run build - Each word of the Header name is in the title case format (i.e. Sec-Fetch-Mode)

**Comment by TheAlexLichter** [comment]
Does the same happen when using plain [Nitro](https://github.com/unjs/nitro)?

**Reply by rickgregg** [reply]
I haven't tried this.
